HelpSource/Classes/Process.schelp

```diff
@@ -24,7 +24,9 @@ Returns the full path to the file containing the code that is currently executin
 
 teletype::nowExecutingPath:: is valid only for interactive code, i.e., code files with a teletype::.scd:: extension. It does not apply to class definitions (teletype::.sc::). For that, use code::thisMethod.filenameSymbol:: or code::this.class.filenameSymbol::.
 
-This method is supported in the SuperCollider IDE, the macOS-only SuperCollider.app, and the scel (SuperCollider-Emacs-Lisp) environment. In other editor environments, it will return code::nil::.
+This method is supported in the SuperCollider IDE, the scel (SuperCollider-Emacs-Lisp) environment and the command-line interface (CLI). In other editor environments, it will return code::nil::.
```

Comment:
> In other editor environments, it will return code::nil::.

I have only tested it with
- SC-IDE on MacOS, Windows and Ubuntu
- Terminal on MacOS and Ubuntu
- Command Prompt on Windows

So I do not know if it is true or not.

Reply:
This edition is already an enhancement since the "macOS-only SuperCollider.app" doesn't exist anymore.

Reply:
nowExecutingPath **does** work on SCNvim on Mac! 

...how about if we write: "This method is supported in the SuperCollider IDE, SCNvim, scel, and the command-line interface (CLI). In unsupported editor environments, it will return code::nil::."

Reply:
@cdbzb Thanks for the suggestion!
After the following thread,

https://scsynth.org/t/what-does-thisprocess-nowexecutingpath-return-in-editor-environments-other-than-sc-ide-scel-and-terminal-command-prompt/9474/4

The following IDE's support this feature: Visual Studio Code, VSCodium and Neovim.

I am not sure, but Neovim seems to be the same as SCNvim.... Am I right?

Combining these with your idea results in:

"This method is supported in the SuperCollider IDE, SCNvim, scel, Neovim, Visual Studio Code, VSCodium, and the command line interface (CLI). In unsupported editor environments it will return code::nil::".

How about that? It is just a suggestion to reflect my little poll on the scsynth forum.

Reply:
Suggestion: 

The method is supported in various environments, including the SuperCollider IDE, SCNvim, Neovim, scel (Emacs), VSCode, VSCodium, and the command line interface (CLI). In unsupported editors, it will return code::nil::.

Reply:
@cdbzb You can come up with a better way to refer to the vim side. SCNvim is a "mode" or an editor?

Reply:
Visual Studio Code and vscode and two different editors. Be careful not to let them confuse you.

Reply:
@smoge 
Thanks, I will use your suggestion!
>The method is supported in various environments, including the SuperCollider IDE, Neovim (SCNvim), Emacs (scel), VSCode/VSCodium, and the command line interface (CLI). In unsupported editors, it will return code::nil::.